src/test/files/DevonUcImplSecurityConstraintCheckCase2.java

```diff
@@ -0,0 +1,15 @@
+package com.devonfw.testapplication.testcomponent.logic.impl;
+
+public class TestClassImpl implements TestClass {
+
+    @Override
+    @DenyAll
```

Comment:
`@RolesAllowed(ApplicationAccessControlConstants.PERMISSION_TEST)`
would be a more realistic example, but fine...

Reply:
If you want to improve here, you can add an additional file with such annotation to improve coverage of the possible cases.